plugins/inputs/procstat/README.md

```diff
@@ -27,8 +27,28 @@ Additionally the plugin will tag processes by their PID (pid_tag = true in the c
 * pid
 * process_name
 
+
+### Windows
```

Comment:
We will have to remember to take a second pass at this documentation, so that it matches the current design